# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/iceauth/hotfix-stdint.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

With the older sparc32 glibc-2.22:
process.c: In function 'reallocarray':
process.c:277:25: error: 'SIZE_MAX' undeclared (first use in this function)

--- iceauth-1.0.10/process.c.vanilla	2024-03-21 18:15:11.812723809 +0100
+++ iceauth-1.0.10/process.c	2024-03-21 18:15:24.244723183 +0100
@@ -32,6 +32,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <signal.h>
+#include <stdint.h>
 
 #define SECURERPC "SUN-DES-1"
 #define K5AUTH "KERBEROS-V5-1"
